datarepo-clienttests/src/main/java/common/utils/AuthenticationUtils.java

```diff
@@ -29,7 +29,7 @@ private AuthenticationUtils() {}
   // the list of scopes we request from end users when they log in. this should always match exactly
   // what the UI requests, so our tests represent actual user behavior
   private static final List<String> userLoginScopes =
-      Arrays.asList("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-platform");
+      Arrays.asList("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-billing");
```

Comment:
May as well update to `List.of()` while you're here
```suggestion
      List.of("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-billing");
```